### PR TITLE
Make input view common and fix ptable accordingly

### PR DESCRIPTION
**Pull request by oshtaier**

### CODE REVIEW

File: airgun/views/partitiontable.py [hunk 1]
```diff
@@ -40,51 +42,48 @@ def is_displayed(self):
...
+    @View.nested
+    class template(SatTab):
+        name = TextInput(id='ptable_name')
+        default = Checkbox(id='ptable_default')
+        snippet = Checkbox(locator="//input[@id='ptable_snippet']")
+        os_family_selection = ConditionalSwitchableView(reference='snippet')
+
+        @os_family_selection.register(True)
+        class SnippetOption(View):
+            pass
```
Comment (ldjebran) [inline airgun/views/partitiontable.py]: Do we still need to register this even if there is no content is this caused by some ConditionalSwitchableView execption?

Comment (oshtaier) [reply airgun/views/partitiontable.py]: hmm, it was there initially. Probably, I can test without

Comment (oshtaier) [reply airgun/views/partitiontable.py]: yeah. That will produce:
```
ValueError: Could not find a corresponding registered view.
```

**Comment by oshtaier** [comment]
https://github.com/SatelliteQE/robottelo/pull/6723 contains test results for all the changes here

**Comment by omkarkhatavkar** [comment]
> ntains test results for all the changes her

I just missed that thanks for updating.

**Pull request merged**